### ISSUE TITLE
IPropertyHandler Set not triggered for BulkInsert operation

### ISSUE THREAD

**Issue by Trobeloo** [mikependon/RepoDB.NET#41]
I have an entity with an enum property that is saved as a string in the database. Because the database string contains space and diacrtitics signs I have created an IPropertyHandler that maps between the string and the enum both for Set and Get.
It works well if I read and write single records but if I use the BulkInsert function the correct string is not saved instead it is the enum part.
Is it normal?

I am using RepoDb.SqlServer.BulkOperations 1.1.6-beta1 on Windows 11 VS 2022.

**Comment by mikependon** [comment]
Yes, this is not triggered on the Bulk Operations yet (both SQL Server and PostgreSQL). This is on our timeline.

**Comment by Trobeloo** [comment]
Ok good to know.
